src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/MockedEndpointsTestBase.kt

```diff
@@ -231,5 +289,50 @@ abstract class MockedEndpointsTestBase : IntegrationTestBase() {
     )
   }
 
+  fun calculateTierFor(crn: String) {
```

Comment:
This has taken a lot of code out of the other tests, which has made them much clearer.